s29986-PrzemyslawKosiedowski/zajecia2/zadanie

```diff
@@ -0,0 +1,17 @@
+W opisie basha (man) przeczytac opis polecenia wewnetrznego umask i wyprobowac
+jego dzialanie sprawdzajac, a nastepnie zmieniajac swoja maske trybu pliku i
+tworzac za kazdym razem nowe pliki (przy uzyciu polecenia touch plik), a
+pozniej sprawdzajac uzyskane uprawnienia dostepu do nich (polecenie ls -l).
+Jaka operacja logiczna na bitach domyslnych uprawnien dostepu oraz maski
+jest wykonywana?
+
+Wykonanie: umask 0022; touch p1; umask 0427; touch p2; umask 0242; touch p3
+Wynik:
+-rw-r--r-- dla p1
+--w-r----- dla p2
+-r---w-r-- dla p3
+Wniosek:
+Polecenie umask ustawia maske, ktora jest uzywana podczas przypisywania upraw-
+nien do tworzonego pliku. W trakcie tworzenia pliku wykonywana jest operacja
+logiczna AND. Domyslnie pliki sa tworzone z dostepem 666, a katalogi 777.
```

Comment:
Czy jesteś pewien, że jest to operacja AND ?
```
umask 777
touch plik
bity: 000

111 111 111 bity pliku
 A   N   D
111 111 111 (umask)
=== === ===
111 111 111 (tak by wyglądała operacja AND). umask tak nie działa.

dla umask 022
111 111 111 bity pliku
 ?   ?   ?
000 010 010 (umask)
=== === ===
111 101 101 dziala tak
```
Zatem jaka tu jest operacja robiona?
Poza tym nie podano źródła skad opinia, że jest to operacja AND.